app/posts/get-an-identity/2022-07-12-names-in-get-an-identity.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Asking for people's names
+date: 2022-07-12
+---
+
+## The problem
+
+Many people typically go by a name that is not the first name listed on their passport, driving license or birth certificate, for example: 
+
+- someone with the official name Jonathan Smith might be called John
+- someone with the official name Claire Sarah Jones might be called Sarah
+
+Others have names which do not fit neatly into the first name / last name distinction, and they may use different variations of their first and last name in different contexts. 
+
+To make life easier for these people, services should be clear when they are asking for a name which needs to match official documents, rather than asking what someone wants to be called. This includes cases where the service will create an official record, or match the user against one, for example:
+- the service involves an identity check
+- the service creates a new record in the Database of Qualified Teachers
+- the service involves checking someone’s record in the Database of Qualified Teachers
+
+## Asking for official name
+
+By "official name" we mean a name that someone can prove with an official document, such as their passport, driving license, or marriage certificate.
+
+Registering for an NPQ with Get an identity involves checking someone’s record in DQT and an identity check (performed by the NPQ provider), so we will specifically ask users for their official name. 
+
+### Users with multiple official names
+
+We know that some people have different names on different official documents, e.g. their name is different on their passport and driving license. We will ask for their most recent official name. 
+
+### Matching official names
+
+Get an identity will check the official name a user provides against their record in the Database of Qualified Teachers. If it matches, then we will store this as a verified official name. 
+
+If it doesn't match, for example because the person has changed their name, then we will store both the official name we found in the DQT, as a verified official name, and the name the user provided, as an unverified official name, treating the unverified name as the more recent one.
+
+Register for an NPQ will have access to both the verified and unverified official names, if present, and can pass whichever is appropriate to providers.
+
+### Updating official names in Get an identity
+
+To have your official name verified in Get an identity, users will need to submit a change request with an identity document which meets the standards used by the TRA as proof. 
```

Comment:
> meets the standards used by the TRA as proof.

It might be helpful to capture what these are here, or link off to where it's documented?